### ISSUE TITLE
It fails, what can i do? not savvy enough.

### ISSUE THREAD

**Issue by RetroGameTalk** [lesaff/serverpilot-letsencrypt#80]
> We're almost done here. Opening HTTPS Port and  Restarting nginx...
> Rule added
> Rule added (v6)
> Job for nginx-sp.service failed because the control process exited with error code. See "systemctl status nginx-sp.service" and "journalctl -xe" for details.

```
:/usr/local/bin# systemctl status nginx-sp.service
● nginx-sp.service - nginx-sp
   Loaded: loaded (/lib/systemd/system/nginx-sp.service; enabled; vendor preset: enabled)
   Active: failed (Result: start-limit-hit) since Tue 2018-11-06 11:59:14 PST; 4s ago
  Process: 29994 ExecStop=/bin/kill -s QUIT $MAINPID (code=exited, status=0/SUCCESS)
  Process: 30045 ExecStartPre=/opt/sp/nginx/sbin/nginx -t (code=exited, status=1/FAILURE)
 Main PID: 727 (code=exited, status=0/SUCCESS)

Nov 06 11:59:13 x06468 systemd[1]: nginx-sp.service: Control process exited, code=exited status=1
Nov 06 11:59:13 x06468 systemd[1]: Failed to start nginx-sp.
Nov 06 11:59:13 x06468 systemd[1]: nginx-sp.service: Unit entered failed state.
Nov 06 11:59:13 x06468 systemd[1]: nginx-sp.service: Failed with result 'exit-code'.
Nov 06 11:59:14 x06468 systemd[1]: nginx-sp.service: Service hold-off time over, scheduling restart.
Nov 06 11:59:14 x06468 systemd[1]: Stopped nginx-sp.
Nov 06 11:59:14 x06468 systemd[1]: nginx-sp.service: Start request repeated too quickly.
Nov 06 11:59:14 x06468 systemd[1]: Failed to start nginx-sp.
Nov 06 11:59:14 x06468 systemd[1]: nginx-sp.service: Unit entered failed state.
Nov 06 11:59:14 x06468 systemd[1]: nginx-sp.service: Failed with result 'start-limit-hit'.
```
`

What does it mean?

**Comment by iantearle** [comment]
I keep getting this on every domain I add now. Whats changed? Something SeverPilot have done?

Though my error is slightly different, I get

Skipping adding existing rule
Skipping adding existing rule (v6)

**Comment by ilearnbydoing** [comment]
yes it is failing, it is no more issuing new certificate or renewing existing one.

**Comment by monsieurPaon** [comment]
Same issue 

● nginx-sp.service - nginx-sp
   Loaded: loaded (/lib/systemd/system/nginx-sp.service; enabled; vendor preset: enabled)
   Active: failed (Result: start-limit-hit) since Fri 2019-07-12 14:10:11 UTC; 44s ago
  Process: 22231 ExecStop=/bin/kill -s QUIT $MAINPID (code=exited, status=0/SUCCESS)
  Process: 22036 ExecStartPost=/bin/sleep 0.1 (code=exited, status=0/SUCCESS)
  Process: 22031 ExecStart=/opt/sp/nginx/sbin/nginx (code=exited, status=0/SUCCESS)
  Process: 22257 ExecStartPre=/opt/sp/nginx/sbin/nginx -t (code=exited, status=1/FAILURE)
 Main PID: 22035 (code=exited, status=0/SUCCESS)

Jul 12 14:10:10 server systemd[1]: nginx-sp.service: Control process exited, code=exited status=1
Jul 12 14:10:10 server systemd[1]: Failed to start nginx-sp.
Jul 12 14:10:10 server systemd[1]: nginx-sp.service: Unit entered failed state.
Jul 12 14:10:10 server systemd[1]: nginx-sp.service: Failed with result 'exit-code'.
Jul 12 14:10:11 server systemd[1]: nginx-sp.service: Service hold-off time over, scheduling restart.
Jul 12 14:10:11 server systemd[1]: Stopped nginx-sp.
Jul 12 14:10:11 server systemd[1]: nginx-sp.service: Start request repeated too quickly.
Jul 12 14:10:11 server systemd[1]: Failed to start nginx-sp.
Jul 12 14:10:11 server systemd[1]: nginx-sp.service: Unit entered failed state.
Jul 12 14:10:11 server systemd[1]: nginx-sp.service: Failed with result 'start-limit-hit'.

**Comment by regiside** [comment]
Oof, Same issue. whole server is down now cause nginx won't start back up. Any hints on how I can get it back online?

**Comment by xyzulu** [comment]
> Oof, Same issue. whole server is down now cause nginx won't start back up. Any hints on how I can get it back online?

https://github.com/lesaff/serverpilot-letsencrypt#if-things-go-wrong

**Comment by iantearle** [comment]
Do a config test. Most likely something wrong in your Nginx configuration

**Comment by RetroGameTalk** [comment]
All details: https://github.com/lesaff/serverpilot-letsencrypt/issues/85
